crates/usvg-parser/src/text.rs

```diff
@@ -8,6 +8,7 @@ use kurbo::{ParamCurve, ParamCurveArclen};
 use svgtypes::{Length, LengthUnit};
 use usvg_tree::*;
 
+use crate::converter::resolve_transform_origin;
```

Comment:
Prefer `use crate::converter::{self, resolve_transform_origin};` 
No need to import `converter` twice. Same with all other instances of `resolve_transform_origin`.